Server/mods/deathmatch/logic/CAccount.h

```diff
@@ -101,6 +101,7 @@ class CAccount
     bool                       AuthorizeSerial(const SString& strSerial, const SString& strWho);
     bool                       RemoveSerial(const SString& strSerial);
     void                       RemoveUnauthorizedSerials();
+    void                       SetAccountSerial(const std::string strSerial);
```

Comment:
Please avoid using Hungarian notation in new code, `serial` would be neater

Reply:
adjusted 9ad1188